component/site/libraries/commonfunctions.php

```diff
@@ -360,8 +360,12 @@ public static function sendAdminMail( $adminName, $adminEmail, $subject='', $tit
 		$messagetemplate = str_replace("{TITLE}", $title,$messagetemplate);
 		$messagetemplate = str_replace("{DESCRIPTION}", $content,$messagetemplate);
 		if ($event){
+			$location = $event->location();
+			if (isset($event->_loc_title)){ $location = $event->_loc_title(); }
 			$messagetemplate = str_replace("{CATEGORY}", $event->catname(),$messagetemplate);
+			$messagetemplate = str_replace("{LOCATION}", $location,$messagetemplate);
```

Comment:
This will only work for sites NOT using managed locations since it will return a numeric value - I have a better fix for the managed locations plugin which I will commit and push.